client/landing/jetpack-cloud/components/log-item/style.scss

```diff
@@ -0,0 +1,47 @@
+.log-item {
```

Comment:
Something seems to be moving weird when expanding and collapsing. Compare the spacing above and below the item when expanded vs when collapsed:

![](https://cldup.com/R3dpU7LI2t.png)

![](https://cldup.com/Z5KKxKQ5uU.png)

Reply:
Yep, that's part of `FoldableCard`. You can verify by checking out the DevDocs.